prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -1,8 +1,9 @@
 import os
 
 from colorama import Fore, Style
-from kubernetes.client.exceptions import ApiException, TimeoutError
+from kubernetes.client.exceptions import ApiException
```

Comment:
Why did you remove the `TimeoutError` from the `kubernetes` API?

Reply:
It was a typo, there is no TimeoutError from kubernetes.client.exceptions